internal/common/tsconfig.bzl

```diff
@@ -161,7 +161,7 @@ def create_tsconfig(ctx, files, srcs,
   # Keep these options in sync with those in playground/playground.ts.
   compiler_options = {
       # De-sugar to this language level
-      "target": "es5" if devmode_manifest or ctx.attr.runtime == "nodejs" else "es2015",
+      "target": "es2015",
```

Comment:
note: this is shared with google3 which we aren't ready to flip yet, so I think this should go in `internal/build_defs.bzl` where we override the results from `tsc_wrapped_tsconfig`

Reply:
done, `internal/common/tsconfig.bzl` is now unchanged